ocr/static/ocr/js/rbi_dummy_table.js

```diff
@@ -3,13 +1,15 @@
+function createdFormatter (value) {
+  return moment(value).format('dddd, MMMM Do YYYY, h:mm:ss:SSS A')
 }
 
 /* eslint no-unused-vars:0 */
-function dateFormatter(value) {
+function dateFormatter (value) {
```

Comment:
Codacy has a fix for the issue: [Unexpected space before function parentheses.](https://app.codacy.com/gh/PrynsTag/oneBarangay/pullRequest?prid=8186851)
```suggestion
function dateFormatter(value) {
```